cfpb/cfgov-refresh
# Secondary nav: only show current section's child pages

Only show the child pages of the currently active section in secondary nav on browse pages

## Changes

- Update secondary nav to only find children of current browse page (or its browse page parent)

## Testing

1. Check out branch
2. Navigate to [a browse page with sibling sections](http://localhost:8000/policy-compliance/guidance/supervision-examinations/appeals-policy/)
3. Verify that only the children of the current section show in nav

## Screenshots

### Current nav
![current](https://user-images.githubusercontent.com/778171/37045482-e6593272-211a-11e8-817d-5caaceb70470.png)


### Updated nav

![updated](https://user-images.githubusercontent.com/778171/37045487-e99683f4-211a-11e8-8688-677da4056e67.png)


## Checklist
